mozconfig

```diff
@@ -0,0 +1,28 @@
+# General configuration options
+ac_add_options --prefix=/app
+
+ac_add_options --with-pthreads
+ac_add_options --disable-crashreporter
+ac_add_options --disable-debug
+ac_add_options --disable-gconf
+ac_add_options --disable-necko-wifi
+ac_add_options --disable-tests
+ac_add_options --disable-updater
+
+ac_add_options --enable-application=mail
+ac_add_options --enable-gio
+ac_add_options --enable-install-strip
+ac_add_options --enable-official-branding
```

Comment:
We need [prior written permission](https://www.mozilla.org/en-US/foundation/trademarks/policy/) from Mozilla in order to enable this setting.

Reply:
I'm assuming you're talking about the
`ac_add_options --enable-official-branding` 
line?

Reply:
That doesn't seem to apply here: "If you compile Mozilla unmodified source code (including code and config files in the installer) and do not charge for it, you do not need additional permission from Mozilla to use the relevant Mozilla Mark(s) for your compiled version."

Reply:
Thanks, I was also going to bring this up from my initial skimming of the document.